### ISSUE TITLE
IncrementOnly Bypass

### ISSUE THREAD

**Issue by AstraxVZ** [gibbed/SteamAchievementManager#360]
im having a problem with the stats, the IncrementOnly extra only allows me to increase the value for 1 but if i increase it more it gives an error, is there a way to bypass the increment only?

**Comment by lhwe** [comment]
Not really, you could modify and recompile SAM to increase and commit the changes over and over, but otherwise IncrementOnly values can't really be changed in mass
